### ISSUE TITLE
Precision is not correctly displayed in Tooltip

### ISSUE THREAD

**Issue by christoph-LSN** [open-sdg/open-sdg#1737]
@MoBosse @brockfanning 

**Please complete the following information** 
[Site repository URL:](https://github.com/christoph-LSN/MT_Site)]
[Data repository URL:](https://github.com/christoph-LSN/IM_Data)]
Open SDG version: [ 2.0.0]

**Describe the bug**

The tooltip shows more decimals then in the precision is configured

Example indicator: https://christoph-lsn.github.io/MT_Site/2-1-5/ (hitting "Prozent")

meta data file: https://github.com/christoph-LSN/IM_Data/edit/develop/meta/2-1-5.md

```
precision:
  - unit: Prozent
    decimals: 1
  - unit: Anzahl
    decimals: 0
```

![image](https://user-images.githubusercontent.com/94443931/178720557-be8f2bd8-07f1-400c-a69d-31bb68edb697.png)

**Expected behavior**
As configured in the precision only one decimal scould be shown

**Additional context**
Hitting "Anzahl" the tooltip shows eg 42 in Year 2014 which schould be 42140

![image](https://user-images.githubusercontent.com/94443931/178722186-a17b259f-e92a-48c1-8ff2-b1566d6c5d5c.png)

**Comment by christoph-LSN** [comment]
@brockfanning  I Still have this tooltip problem is there any progress in here?

you can also see it in this indicator: [https://christoph-lsn.github.io/MT_Site/1-2-4/](https://christoph-lsn.github.io/MT_Site/1-2-4/)

only two or three digits are shown in the tooltip but it schould be five or six as you can see in the table

![image](https://user-images.githubusercontent.com/94443931/185634062-5b6fea91-3d83-4e14-b87b-97a8e8ab514c.png)

mf file: [https://github.com/christoph-LSN/IM_Data/blob/develop/meta/1-2-4.md](https://github.com/christoph-LSN/IM_Data/blob/develop/meta/1-2-4.md)

**Comment by MoBosse** [comment]
@christoph-LSN I just ran into a different problem ( #1796) related to configuring the value format in the tooltips. Only now I noticed that I have the same situation as you. The values in the chart tooltip do not seem to be affected by the defined number of decimal places.
It's wired somehow, as the values in the table (including data from target annotations) are formatted as expected. Unless I'm very much mistaken, the same function is used for the table values and the tooltip values....
